Dockerfile.windows

```diff
@@ -262,29 +262,16 @@ RUN `
 RUN `
   Function Build-GoTestSum() { `
     Write-Host "INFO: Building gotestsum version $Env:GOTESTSUM_COMMIT in $Env:GOPATH"; `
-    $optsForGet = @('"get"', '"-d"', '"gotest.tools/gotestsum"'); `
-    &go $optsForGet; `
-    $savedExitCode = $LASTEXITCODE; `
-    if ($savedExitCode -ne 0) {  `
+    $env:GO111MODULE = 'on'; `
+    &go get -d "gotest.tools/gotestsum@${Env:GOTESTSUM_COMMIT}"; `
+    $env:GO111MODULE = 'off'; `
```

Comment:
this should go after LASTEXITCODE

Reply:
ah, probably need to add an intermediate variable again 😞

Reply:
not sure actually; can `$env:FOO = something` fail?

Reply:
Looks like it's not needed;

```powershell
PS> go get -d example.com/foobar
unrecognized import path "example.com/foobar": reading https://example.com/foobar?go-get=1: 404 Not Found
PS> echo $LASTEXITCODE
1
PS> $env:GO111MODULE = 'off';
PS> echo $LASTEXITCODE
1
```